packages/inventory/src/Stocktake/DetailView/reducer.ts

```diff
@@ -121,6 +170,37 @@ export const reducer = (
           state.draft.status = action.payload.newStatus;
           break;
         }
+
+        case StocktakeActionType.SortBy: {
+          const { payload } = action;
+          const { column } = payload;
+
+          const { key } = column;
+
+          const { draft, sortBy } = state;
+          const { lines } = draft;
+          const { key: currentSortKey, isDesc: currentIsDesc } = sortBy;
+
+          const newIsDesc = currentSortKey === key ? !currentIsDesc : false;
+          const newDirection: 'asc' | 'desc' = newIsDesc ? 'desc' : 'asc';
+          const newSortBy: SortBy<StocktakeItem> = {
+            key,
+            isDesc: newIsDesc,
+            direction: newDirection,
+          };
+
+          const sortedLines = lines.sort(
```

Comment:
the sorting isn't working for me - it looks like it should. not sure if I should be waiting for another part of the PR or not?

Reply:
mm no, I wasn't sure what to do about the `[multiple]` values and sorting so I've left for now 🤔